src/components/testCaseList/TestCase.tsx

```diff
@@ -22,6 +24,26 @@ const TestCase = (props) => {
   const status = testCase.executionStatus;
   const statusColor = testCase.executionStatus === "pass" ? "success" : "error";
 
+  let titleLength = 0;
+  let titleNeedTruncate = false;
+  let displayTitle = testCase.title;
+  if (testCase.title != null && testCase.title.trim() !== "") {
+    titleLength = testCase.title.length;
+    titleNeedTruncate = titleLength > 60 ? true : false;
+    displayTitle =
+      titleLength < 60 ? displayTitle : displayTitle.substring(0, 59);
+  }
+
+  let seriesLength = 0;
+  let seriesNeedTruncate = false;
+  let displaySeries = testCase.series;
+  if (testCase.series != null && testCase.series.trim() !== "") {
+    seriesLength = testCase.series.length;
+    seriesNeedTruncate = seriesLength > 60 ? true : false;
+    displaySeries =
+      seriesLength < 60 ? displaySeries : displaySeries.substring(0, 59);
+  }
```

Comment:
probably better to create a dumb component that truncates any given text and reuse it wherever needed

Reply:
@adongare That's a great idea! Thanks a lot I'll work on it!

Reply:
@adongare Hi Ashok! I see you approve the pull request, is it ok if I click the Resolve conversion and merge the PR? Thanks so much for your constant help!!

Reply:
Sorry, yes. forgot to resolve this.